### ISSUE TITLE
Red window blocking tool 

### ISSUE THREAD

**Issue by Destiny-123** [PolarBean/DLC_ROI_tool#19]
Hello, 

I am working on a mac and have faced the of issue when I launch the tool and try to outline an ROI the screen goes completely red and blocks the selected frame. The only way thing that shows up is when I set and name a ROI but I cannot see myself drawing the ROI.

**Comment by PolarBean** [comment]
make sure you have the correct version of the packages described in 
https://github.com/PolarBean/DLC_ROI_tool/blob/master/ROI_ENV.yml
Preferably install directly with conda

**Comment by Destiny-123** [comment]
Yes, everything is updated. I even deleted and reinstalled and still have the same issues.
